# An example configuration that works with the example
# 
# It is also used in the integration tests

CartesianImpedance_trajectory_controller:
  type: cartesian_impedance_controller/CartesianImpedanceController
  joints:                               # Joints to control
    - joint1
    - joint2
    - joint3
    - joint4
    - joint5
    - joint6
 
  end_effector: link8                   # Link to control arm in
  update_frequency: 100                 # Controller update frequency in Hz
  dynamic_reconfigure: true             # Starts dynamic reconfigure server
  handle_trajectories: true             # Accept traj., e.g. from MoveIt
  robot_description: /robot_description # In case of a varying name
  wrench_ee_frame: link8_ee_fk          # Default frame for wrench commands
  delta_tau_max: 0.1                    # Max. commanded torque diff between steps in Nm
  filtering:                            # Update existing values (0.0 1.0] per s
    nullspace_config: 0.1               # Nullspace configuration filtering
    pose: 0.1                           # Reference pose filtering
    stiffness: 0.1                      # Cartesian and nullspace stiffness
    wrench: 0.1                         # Commanded torque
  verbosity:
    verbose_print: false                # Enables additional prints
    state_msgs: true                    # Messages of controller state
    tf_frames: true                     # Extra tf frames

joint_state_controller:
    type: joint_state_controller/JointStateController
    publish_rate: 100

# Settings for ros_control hardware interface
hardware_interface:
  control_freq: 100 # in Hz
  joints:
    - joint1
    - joint2
    - joint3
    - joint4
    - joint5
    - joint6
    
